src/cpu/aarch64/acl_eltwise.hpp

```diff
@@ -74,11 +74,17 @@ struct acl_eltwise_fwd_t : public primitive_t {
             using namespace data_type;
             const memory_desc_wrapper data_d(data_md());
 
-            bool ok = is_fwd() && one_of(data_d.data_type(), f32, s32, s8)
+            bool ok = is_fwd() && one_of(data_d.data_type(), f32, f16, s32, s8)
                     && !has_zero_dim_memory() && attr()->has_default_values()
                     && data_d.is_dense();
             if (!ok) return status::unimplemented;
 
+            // Workaround for the nan-value caused by tanh of ACL-FP16 int gtest(test_eltwise).
```

Comment:
It would be worth raising this as an issue against ACL I think

Reply:
You are right, I will code a program that can reproduce this bug for ACL later.

Reply:
https://github.com/ARM-software/ComputeLibrary/issues/998